tests/loggen/loggen.c

```diff
@@ -269,13 +274,32 @@ init_csv_statistics(void)
     }
 }
 
-static void
+static int
 start_plugins(GPtrArray *plugin_array)
 {
   if (!plugin_array)
     {
       ERROR("invalid reference for plugin_array\n");
-      return;
+      return 0;
+    }
+
+  /* check plugins to see how many is activated by command line parameters */
+  int number_of_active_plugins = 0;
+  for (int i=0; i < plugin_array->len; i++)
+    {
+      PluginInfo *plugin = g_ptr_array_index(plugin_array,i);
+      if (!plugin)
+        continue;
+
+      if (plugin->is_plugin_activated())
+        number_of_active_plugins++;
+    }
+
+  if (number_of_active_plugins != 1)
+    {
+      ERROR("%d plugin activated. You should activate exactly one plugin at a time. See \"loggen --help-all\" for available plugin options\n",
```

Comment:
`%d plugins activated.`, because that's correct for both 0 and >1

Reply:
added the 's'